autogpt_platform/frontend/src/components/NodeHandle.tsx

```diff
@@ -53,10 +61,10 @@ const NodeHandle: FC<HandleProps> = ({
           type="target"
           position={Position.Left}
           id={keyName}
-          className="background-color: white; border: 2px solid black; width: 15px; height: 15px; border-radius: 50%; bottom: -7px; left: 20%; group -ml-[26px]"
+          className="background-color: white; border: 2px solid black; width: 15px; height: 15px; border-radius: 50%; bottom: 0px; left: 20%; group -ml-[26px]"
```

Comment:
Is pure css in a classname even valid? should it be removed? 
The fact that it's changed is this might change something, but I'm super curious now.

Reply:
I don't know, it seems to work. Defo should prob move it to tailwind

Reply:
Confirmed doesn't work, removed it